documentation/reference/services-content.html

```diff
@@ -284,7 +284,9 @@ <h2 id="document-processing">document-processing</h2>
 <h2 id="redundancy">redundancy</h2>
 <p>
 Contained in <code><a href="#content">content</a></code>.
-Defines the total number of copies of each piece of data the cluster will maintain to avoid data loss.
+Defines the total number of copies of each piece of data the cluster will maintain to avoid data loss
+- within a <a href="#group">group</a>.
+If <em>group</em> is not configured, this equals one group, hence all nodes.
```

Comment:
Doesn't group-aware auto scaling of redundancy only apply iff you're running in a hosted setting?

Reply:
don't know ...

Reply:
Yes, "within a group" is only true on hosted Vespa.